core/api/service/base_request.hpp

```diff
@@ -40,10 +40,14 @@ namespace kagome::api::chain::request {
   }                                                                       \
   void loadValue(type &dst, const jsonrpc::Value &src)
 
-  template <typename ResultType, typename... Types>
+  template <typename ResultType, typename... ArgumentTypes>
   struct RequestType {
+   public:
+    using Params = std::tuple<ArgumentTypes...>;
```

Comment:
This tuple is the private and was encapsulated. Why it's type become public?

Reply:
It makes types set acessible to using.  It's not about values, whicht stays private.